javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -240,7 +239,7 @@ export async function generateNetworkSpec(
       const isCumulusBased =
         parachain.cumulus_based !== undefined
           ? parachain.cumulus_based
-          : ![DEFAULT_ADDER_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(
+          : ![DEFAULT_COLLATOR_IMAGE, UNDYING_COLLATOR_BIN].includes(
```

Comment:
```suggestion
          : ![DEFAULT_ADDER_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(
```
We should check agains the adder-collator command.

Reply:
Ah sorry. This was erroneously altered.. Thank you!